doc/modules/model_evaluation.rst

```diff
@@ -2711,6 +2711,80 @@ Here are some usage examples of the :func:`d2_absolute_error_score` function::
   >>> d2_absolute_error_score(y_true, y_pred)
   0.0
 
+.. _visualization_regression_evaluation:
+
+Visual evaluation of regression models
+--------------------------------------
+
+Among methods to assess the quality of regression models, scikit-learn provides
+the :class:`~sklearn.metrics.PredictionErrorDisplay` class. It allows to
+visually inspect the prediction errors of a model in two different manners.
+
+.. image:: ../auto_examples/model_selection/images/sphx_glr_plot_cv_predict_001.png
+   :target: ../auto_examples/model_selection/plot_cv_predict.html
+   :scale: 75
+   :align: center
+
+The plot on the left shows the actual values vs predicted values. For a
+noise-free regression task aiming to predict the (conditional) expectation of
+`y`, a perfect regression model would display data points on the diagonal
+defined by predicted equal to actual values. The further away from this optimal
+line, the larger the error of the model. In a more realistic setting with
+irreducible noise, that is, when not all the variations of `y` can be explained
+by features in `X`, then the best model would lead to a cloud of points densely
+arranged around the diagonal.
+
+Note that the above only holds when the predicted values is the expected value
+of `y` given `X`. This is typically the case for regression models that
+asymptotically minimize the mean squared error objective function or more
+generally the :ref:`mean Tweedie deviance <mean_tweedie_deviance>` for any
+value of its "power" parameter.
+
+When plotting the predictions of an estimator that predicts a quantile
+of `y` given `X`, e.g. :class:`~sklearn.linear_model.QuantileRegressor`
+or any other model asymptotically minimizing the :ref:`pinball loss
+<pinball_loss>`, a fraction of the points are either expected to lie above or
+below the diagonal depending on the estimated quantile level.
+
+All in all, while intuitive to read, this plot does not really inform us on
+what to do to obtain a better model.
+
+The right-hand side plot shows the residuals (i.e. the difference between the
+actual and the predicted values) vs. the predicted values.
+
+This plot makes it easier to visualize if the residuals follow and
+`homoscedastic or heteroschedastic
+<https://en.wikipedia.org/wiki/Homoscedasticity_and_heteroscedasticity>`_
+distribution.
+
+In particular, if the true distribution of `y|X` is Poisson or Gamma
+distributed, it is expected that the variance of the residuals of the optimal
+model to grow with the predicted value of `E[y|X]` (either linearly for
+Poisson or quadratically for Gamma).
+
+Since a predictive models do not adapt themselves to the particularities of the
+true distribution, it comes to us at choosing the optimal modelisation given
+the data at hand. This plot is therefore useful for making a choice between
+different family of models.
+
```

Comment:
I don't think the first sentence is true. Any non-trivial machine learning model
adapts its parameters to best match the empirical distribution.

But even if you refer to shape of the true conditional distribution and try to match it by choosing different fitting criterions or parametrizations, it would still be possible to design a metaestimator that runs and an internal CV and tries many estimator classes.

Anyways, I don't think this paragraph brings any useful information beyond what is explained in more specific previous and following paragraphs. Instead it can bring confusion so I would suggest to just drop it.

```suggestion
```